experiments/main_agentic/run.py

```
@@ -0,0 +1,1422 @@
+    valid = _normalized_verifier_valid(normalized)
+    if isinstance(valid, bool):
+        normalized["valid"] = valid
+    return normalized
+
+
+def _run_satnet_verifier_api(case_dir: Path, solution_path: Path) -> VerifierOutcome:
+    try:
+        from benchmarks.satnet.verifier import verify_case
+
+        result = verify_case(case_dir, solution_path)
+    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)